src/awesome/lua/utils.rs

```diff
@@ -29,6 +29,24 @@ pub fn mods_to_lua<'lua>(lua: &'lua Lua, mods: &[Key]) -> rlua::Result<Table<'lu
     lua.create_table_from(mods_list.into_iter().enumerate())
 }
 
+/// Convert a single number to a modifier list.
+pub fn num_to_mods(modifiers: KeyboardModifier) -> Vec<Key> {
+    let mut res = vec![];
+    let mod_types = [(KeyboardModifier::WLR_MODIFIER_SHIFT, KEY_Shift_L),
```

Comment:
Is it possible to make this a global constant? I have a feeling we will use this again somewhere.